lab3/src/pages/dashboard/calendar/index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Main Page</title>
+    <link rel="stylesheet" href="/styles/index.css" />
+  </head>
+  <body>
+    <div class="dashboard">
+      <div class="sidebar" id="sidebar">
+        <img src="/assets/images/logo.png" alt="" class="sidebar__logo" />
+
+        <div class="sidebar-item my-4" id="sidebar-toggle">
+          <div class="sidebar-item__text"></div>
+          <img
+            src="/assets/icons/sidebar/open.svg"
+            alt=""
+            class="sidebar-item__icon"
+          />
+        </div>
+        <a class="sidebar-item" data-section="home" href="/dashboard">
+          <img
+            src="/assets/icons/sidebar/home.svg"
+            alt=""
+            class="sidebar-item__icon"
+          />
+          <div class="sidebar-item__text">ГЛАВНАЯ</div>
+        </a>
+        <div class="sidebar-item" data-section="classes">
```

Comment:
This should also be clickable?

Reply:
same here, if smth is happened and ur js will be disabled, ur page should still has the working skeleton. If it's redirecting to another page than semantically it should be `a`, later in js u can correct the behaviour of redirection with preventDefault. The behaviour of this elements does not require from u any specific behaviour. If ScreenReader gonna read ur `<div>`s it will die.

Reply:
ok

Reply:
fixed